.gitignore

```diff
@@ -1,2 +1,160 @@
-*.pyc
-*~
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
```

Comment:
Is this always meant to be an automatically-generated file?

Reply:
Based on [python documentation](https://docs.python.org/3/distutils/sourcedist.html) - yes:

```
The manifest template is just a list of instructions for how to generate your manifest file, MANIFEST, which is the exact list of files to include in your source distribution. The sdist command processes this template and generates a manifest based on its instructions and what it finds in the filesystem.
```

Reply:
Ah yes thanks, I think I confused it with the manifest template.